uint32 time_stamp    # 设备数据包内包含的时间戳
uint64 tag_number    # 二维码信息
float32 x_deviation  # 毫米
float32 y_deviation  # 毫米
float32 theta        # 度
uint16 error_code    # 错误码， 65535: 离线; 65534: 数据错误